### PR TITLE
github/issue_template: add note about --profile=fast

### PR DESCRIPTION
**Pull request by kasper93**

### CODE REVIEW

**Comment by Hrxn** [comment]
Maybe also mention the `Ctrl+H` shortcut there in these issue templates?

**Comment by hooke007** [comment]
Ctrl+h

File: .github/ISSUE_TEMPLATE/2_bug_report_linux.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ Releases are listed here: https://github.com/mpv-player/mpv/releases
 Try to reproduce your issue with --no-config first. If it isn't reproducible
 with --no-config try to first find out which option or script causes your issue.
 
+If the issue is performance-related, try to reproduce it with --no-config
+--profile=fast, which is designed to work on lower-end mobile devices.
+Additionally, enable hardware decoding with --hwdec=yes or the CTRL+H shortcut.
```
Comment (Dudemanguy) [inline .github/ISSUE_TEMPLATE/2_bug_report_linux.md:31]: Nit but shouldn't it be `Ctrl+h`

Comment (kasper93) [reply .github/ISSUE_TEMPLATE/2_bug_report_linux.md]: done

**Review by Dudemanguy — APPROVED** [review]

**Pull request merged**